website/google.erb

```diff
@@ -434,6 +434,10 @@
       <a href="/docs/providers/google/r/storage_bucket_acl.html">google_storage_bucket_acl</a>
       </li>
 
+      <li<%= sidebar_current("docs-google-storage-bucket-iam") %>>
+      <a href="/docs/providers/google/r/storage_bucket_iam.html">google_storage_bucket_iam_*</a>
```

Comment:
Shouldn't we have docs for member, too?

Reply:
The documentation for `_member` and `_binding` is on the same page. Same format than #814

Reply:
Ah, ok, I missed a trick here. Sorry. Would it be possible to add both resource names to the sidebar, so people looking for the resource don't get confused? They can link to the same file, I'm ok with that, but ideally a Ctrl+F for the resource name should pull up a link on the sidebar.

Reply:
Done. We may want to revisit this later. The goal was to avoid the explosion of resources in the sidebar. At some point, almost every resources will have 2-3 iam resources...

Reply:
Fair. I think there's some ongoing work to fix the sidebar situation, and depending on what that looks like, a refactor may be in order (e.g., if there's collapsible grouping, maybe just have an IAM grouping).

Reply:
👍